api/validate.js

```diff
@@ -0,0 +1,34 @@
+
+const {REACT_APP_GAME_DB_USER, REACT_APP_GAME_DB_HOST, REACT_APP_GAME_DB_NAME, REACT_APP_GAME_DB_PASSWORD, REACT_APP_GAME_DB_PORT} = process.env;
+
+const Pool = require("pg").Pool;
+const pool = new Pool({
```

Comment:
In order to not introduce too many new env vars can we re-use the existing `PANTHEON_PGURI` env var? This would be a connection string in the format of `postgres://user:password@host:5432/database`. You can rewrite this line like so:

```
const pool = new Pool({
  connectionString?: string, // e.g. postgres://user:password@host:5432/database
});
```